boardStupid.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=boardStupid', 'root', 'password');
+$query = $db->prepare('QUERY;');
+$query->execute();
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare('SELECT * FROM `boardStupid`;');
```

Comment:
I think Ash said it was better practice to name the specific fields you are importing rather than using *